data/json/recipes/recipe_others.json

```diff
@@ -4666,5 +4666,17 @@
     "autolearn": true,
     "qualities": [ { "id": "HAMMER", "level": 2 }, { "id": "SAW_W", "level": 2 } ],
     "components": [ [ [ "2x4", 4 ] ], [ [ "nail", 20 ] ], [ [ "hinge", 2 ] ] ]
+  },
+  {
+    "type": "recipe",
+    "result": "stethoscope",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_TOOLS",
+    "skill_used": "fabrication",
+    "difficulty": 1,
+    "time": "15 m",
+    "autolearn": true,
+    "qualities": [ { "id": "CUT", "level": 1 } ],
+    "components": [ [ [ "duct_tape", 30 ] ], [ [ "bottle_plastic_small", 2 ] ], [ [ "hose", 2 ] ] ]
```

Comment:
What about the funnel? We have several types in-game, no reason to not have them as alternatives in the recipe.

Also, why 2 rubber hoses rather than one?

Reply:
> 
> 
> What about the funnel? We have several types in-game, no reason to not have them as alternatives in the recipe.
> 
> Also, why 2 rubber hoses rather than one?

Since there's no "balloon" item in the game, I figured a second hose could be cut and pulled over the other end to provide the membrane. Didn't even think of the condom. Definitely makes more sense. Will use that instead.

I think I'll add the funnel as an alternative to the small bottles.